ui/app/mirrors/create/cdc/schemabox.tsx

```diff
@@ -125,17 +123,16 @@ const SchemaBox = ({
     e: React.MouseEvent<HTMLInputElement, MouseEvent>,
     schemaName: string
   ) => {
-    setHandlingAll(true);
```

Comment:
removed `setHandlingAll` because this function is synchronous. If this should wait for resolution of `addTableColumns/removeTableColumns` then their promises need to be collected & then `new Promise(promises).then(() => setHandlingAll(false))`